templates/core/fields/field--paragraph--field-author.html.twig

```diff
@@ -0,0 +1,72 @@
+{#
```

Comment:
deze velden zijn afhankelijk van een bouwblok, ik zou ze in de contrib folder steken bv.: contrib/parapgrahs/quote/fields/xxx.html.twig

Reply:
Done.